packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/DefaultTable.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Table from 'terra-table';
+
+const tableData = {
+  cols: [
+    { id: 'Column-0', displayName: 'Patient', isResizable: true },
+    { id: 'Column-1', displayName: 'Location' },
+    { id: 'Column-2', displayName: 'Illness Severity' },
+    { id: 'Column-3', displayName: 'Visit' },
+    { id: 'Column-4', displayName: 'Allergy' },
+    { id: 'Column-5', displayName: 'Primary Contact' },
+    { id: 'Column-6', displayName: 'Generic Order Counts' },
+    { id: 'Column-7', displayName: 'Patient Age' },
+    { id: 'Column-8', displayName: 'Medication History' },
+    { id: 'Column-9', displayName: 'My Relationship' },
+    { id: 'Column-10', displayName: 'Not Selectable' },
+  ],
+  rows: [
+    {
+      id: '1',
+      cells: [
+        { content: 'Fleck, Arthur' },
+        { content: '1007-MTN' },
+        { content: 'Unstable' },
+        { content: 'Inpatient, 2 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+        { isMasked: true },
```

Comment:
Table examples shouldn't include masked cells. That feature should only be advertised for Worklist Data Grid unless a need for masking in ordinary tables is established.